src/utbetalinger/UtbetalingerPanel.tsx

```diff
@@ -71,48 +70,41 @@ const UtbetalingerPanel: React.FC<Props> = ({utbetalinger, lasterData}) => {
                                                 tittel={"Utbetalt " + formatDato(utbetalingMaaned.utbetalingsdato)}
                                                 defaultOpen={erDevMiljo()}
                                             >
-                                                <br />
-
-                                                <div>
+                                                <div className="mottaker__wrapper">
                                                     <EtikettLiten>Mottaker</EtikettLiten>
-                                                    {!annenMottaker && (
-                                                        <Element>
-                                                            Til deg (
-                                                            {utbetalingMaaned.utbetalingsmetode && (
-                                                                <>{utbetalingMaaned.utbetalingsmetode} </>
-                                                            )}
-                                                            {utbetalingMaaned.kontonummer})
-                                                        </Element>
-                                                    )}
-                                                    {annenMottaker && (
+                                                    {annenMottaker ? (
                                                         <Element style={{textTransform: "capitalize"}}>
+                                                            annenmottaker
```

Comment:
Skal denne teksten vises? `annenmottaker`

Reply:
Ups! Den var for debugging.. samme med linje 87 🙈

Reply:
Fort gjort, ingen skade skjedd ^^